python/pyspark/sql/functions.py

```diff
@@ -468,6 +468,18 @@ def input_file_name():
     return Column(sc._jvm.functions.input_file_name())
 
 
+@since(2.4)
+def isinf(col):
```

Comment:
Shall we expose this to column.py too?

Reply:
Do you want me to add the function to column.py?

Reply:
Yes, please because I see it's exposed in Column.scala.

Reply:
Added it in my latest commit